spec/higher_level_api/integration/connection_recovery_spec.rb

```diff
@@ -10,7 +10,7 @@
 
   def close_all_connections!
     # wait for stats to refresh, make sure to run bin/ci/before_build.sh as well!
-    sleep 0.7
+    sleep 1.7
```

Comment:
I had huge issues in getting 2 specs here to work (l 318 & 333) increasing this made them stable for me

Reply:
[This script](https://github.com/ruby-amqp/march_hare/blob/master/bin/ci/before_build.sh#L17) updates management plugin settings so that changes are observable over HTTP API earlier. Was it executed?

Reply:
I did run the script. Although I had to modify it to work with the rabbitmq I have running inside a podman/docker container. I can revert and see if it works on travis, would be fine for me. I also didn't try lots of values here just added 1 extra seconds and then the specs worked stable locally.

Reply:
It's fine. Let's keep the value that works for you.